projects/distributed-tracing/src/pages/trace-detail/logs/trace-logs.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { LogEventsTableViewType } from '../../../shared/components/log-events/log-events-table.component';
+import { LogEvent } from '../../../shared/dashboard/widgets/waterfall/waterfall/waterfall-chart';
+import { TraceDetailService } from './../trace-detail.service';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <ng-container *ngIf="this.logEvents$ | async as logEvents">
```

Comment:
If we are fetching from graphql then we should use `*htLoadAsync` so that we can show the loader icon

Reply:
Actually, for this
Data is already being fetched before this, on the page level. and here it will be only used as a static data observable (`shareReplay`)

Reply:
okay. I hope that doesn't change :)

Reply:
It is better to use loadAsync. If in case fetchLogEvents starts returning a new observable, we would have to remember to update this template as well.